src/signals/incident/definitions/wizard-step-2-vulaan/straatverlichting-klokken.ts

```diff
@@ -57,18 +57,9 @@ export const straatverlichtingKlokken = {
       ifAllOf: {
         subcategory: 'lantaarnpaal-straatverlichting',
       },
-      ifOneOf: {
-        extra_straatverlichting_probleem: [
-          'lamp_doet_het_niet',
-          'lamp_brandt_overdag',
-          'geeft_lichthinder',
-          'lamp_is_zichtbaar_beschadigd',
-          'overig',
-        ],
-      },
       wfsFilter:
-        '<BBOX><gml:Envelope srsName="{srsName}"><lowerCorner>{west} {south}</lowerCorner><upperCorner>{east} {north}</upperCorner></gml:Envelope></BBOX>',
-      endpoint: configuration.map.layers?.klokken,
+        '<PropertyIsNotEqualTo><ValueReference>objecttype_omschrijving</ValueReference><Literal>Klok</Literal></PropertyIsNotEqualTo><BBOX><gml:Envelope srsName="{srsName}"><lowerCorner>{west} {south}</lowerCorner><upperCorner>{east} {north}</upperCorner></gml:Envelope></BBOX>',
+      endpoint: configuration.map.layers?.verlichting,
       featureTypes: [
```

Comment:
Why is this in this PR?

Reply:
Ah, probably forgot to revert the change when I came across those wrongly configured settings. I'll make a different PR for this.